php/wordpress/02-customizing/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM cgr.dev/chainguard/wordpress:latest-dev
+ARG UID=1000
+
+USER root
+RUN apk add shadow
```

Comment:
Why is this needed? Should it be in the base image?

Reply:
it's for the `useradd` command! without this user is pretty hard to make volumes work. I have considered adding this user to the -dev image only, but it was a bit complicated to achieve that only for the -dev variant.

Reply:
Isn't `adduser` there already?

Reply:
ooooh gosh, I didn't know that :D updating it now